src/pages/index.tsx

```diff
@@ -1,9 +1,15 @@
+import { Button } from '@nextui-org/react';
+
 export default function Home() {
   return (
     <div>
-      <h1 className="text-3xl font-bold underline">
-        Hello from Developers India
-      </h1>
+      <h1 className="text-3xl font-bold underline">Hello Devs</h1>
```

Comment:
I would also like to add, there's an merge conflict on `package.json` blocking the PR. Since, PostCSS is a TailwindCSS dependency, removing that will also unblock the PR. So, if we can reach an consensus here then #123 can be closed.

Reply:
Tailwind CSS and Next UI are two different things, we can remove either. tailwind is a CSS library that gives you classes to work and make things easy so that you don't have to write all your CSS.

Next UI is a component library it gives you the components out of the box that would be symmetric in the whole project.
There is no complexity being added here, IMO, these are like basic things that we do daily in frontent.

Reply:
Yup, we can remove TailwindCSS then in favour of NextUI.

I just have one little question though. The classes, `text-3xl`, `font-bold` & so on, are they TailwindCSS classes or from NextUI?

Reply:
those classes are from tailwind and the button component from Next UI

Reply:
We'll be removing TailwindCSS in favour of NextUI just so you know. Since the `Button` component uses TailwindCSS classes the website will break.

Reply:
then how will you style components tell me

Reply:
Next UI is a component library, you can style them using basic CSS, but that is a hectic task.
Tailwind CSS gives you classes that will eventually convert to vanilla CSS.

Reply:
We will need both Next UI and Tailwind CSS @Jarmos-san

Reply:
> We will need both Next UI and Tailwind CSS @Jarmos-san

Cool, let's keep it open & review this decision for later. But I don't over-complicating the task at hand is the way to go when we don't even have an MVP in place.